scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/validator/SpelValidatorTest.java

```diff
@@ -44,19 +45,40 @@
 
 public class SpelValidatorTest {
 
+    private StandardEvaluationContext context;
+
+    @Before
+    public void before() {
+        context = new StandardEvaluationContext();
+        context.setTypeLocator(new ModelValidatorContext.RestrictedTypeLocator());
+    }
+
     @Test
     public void testSpelOK() throws ValidationException {
         SpelValidator validator = new SpelValidator("#value == 'MyString'");
         String value = "MyString";
-        Assert.assertEquals(value,
-                            validator.validate(value, new ModelValidatorContext(new StandardEvaluationContext())));
+        Assert.assertEquals(value, validator.validate(value, new ModelValidatorContext(context)));
     }
 
     @Test(expected = ValidationException.class)
     public void testSpelKO() throws ValidationException {
         SpelValidator validator = new SpelValidator("#value == 'MyString'");
         String value = "MyString123";
-        validator.validate(value, new ModelValidatorContext(new StandardEvaluationContext()));
+        validator.validate(value, new ModelValidatorContext(context));
+    }
+
+    @Test(expected = ValidationException.class)
+    public void testSpelUnauthorizedType() throws ValidationException {
+        SpelValidator validator = new SpelValidator("T(Runtime).exec('hostname').waitFor() instanceof T(Integer)");
```

Comment:
actually there was a mistake in this string, I should have used T(java.lang.Runtime).getRuntime().exec( ...